lib/change_env_paths.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var path = require('path');
+var fileSearch = require('./file_search');
+
+function changeEnvPaths(opts, configNameSearch, searchPaths) {
+  var cwd, configPath, configBase;
+
+  if (opts.cwd) {
+    cwd = path.resolve(opts.cwd);
+    if (opts.configPath) {
+      configPath = path.resolve(opts.configPath);
+    } else {
+      searchPaths = [cwd];
+      configPath = fileSearch(configNameSearch, searchPaths);
+    }
+    if (configPath) {
+      configBase = path.dirname(configPath);
+    }
+
+  } else if (opts.configPath) {
+    configPath = path.resolve(opts.configPath);
+    configBase = path.dirname(configPath);
+    cwd = configBase;
+
+  } else {
+    cwd = process.cwd();
+    searchPaths = [cwd].concat(searchPaths);
+    configPath = fileSearch(configNameSearch, searchPaths);
+    if (configPath) {
+      configBase = path.dirname(configPath);
+      cwd = configBase;
+    }
+  }
+
```

Comment:
Sorry, I forgot to move [this part](https://github.com/js-cli/js-liftoff/blob/master/lib/find_config.js#L21-L24) in the old codes here.